charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml

```diff
@@ -13,6 +13,8 @@ data:
   adot-config:  |
     extensions:
       health_check: {{ .Values.adotCollector.daemonSet.extensions.healthCheck }}
+      sigv4auth:
+        region: {{ .Values.awsRegion }}
```

Comment:
Why is the region value here different than the region value used in the removed `awsprometheusremtoewrite` config? 
```
aws_auth:
  region: {{ .Values.adotCollector.daemonSet.ampexporters.region }}
```
Are they always the same region? If so, can the above value be removed from `values.yaml`?

Reply:
Yes, the above value is removed from values.yaml. and we use `awsRegion` which we pass through at the time of install.